### PR TITLE
Code coverage: Org service - overall %

### PR DESCRIPTION
**Pull request by Dimple16**
Screenshot 2023-02-16 at 9 36 38 PM

Note: Ignoring a `catchError` coverage

### CODE REVIEW

File: src/app/core/services/org.service.spec.ts [hunk 1]
```diff
@@ -86,4 +88,54 @@ describe('OrgService', () => {
       done();
     });
   });
+
+  it('switchOrg(): should switch org', (done) => {
+    const orgId = 'orNVthTo2Zyo';
+    apiService.post.and.returnValue(of(apiEouRes));
+    spyOn(globalCacheBusterNotifier, 'next');
```
Comment (rvab) [inline src/app/core/services/org.service.spec.ts:95]: we aren't using it no?

File: src/app/core/services/org.service.spec.ts [hunk 2]
```diff
@@ -86,4 +88,54 @@ describe('OrgService', () => {
...
+
+      orgService.suggestOrgCurrency().subscribe((res) => {
+        expect(res).toEqual('USD');
+        expect(apiService.get).toHaveBeenCalledOnceWith('/currency/ip');
+        done();
+      });
+    });
+  });
+
+  it('setCurrencyBasedOnIp(): should set currecny based on ip', (done) => {
```
Comment (rvab) [inline src/app/core/services/org.service.spec.ts:128]: `currency`

File: src/app/core/services/org.service.spec.ts [hunk 3]
```diff
@@ -86,4 +88,54 @@ describe('OrgService', () => {
...
+        expect(apiService.get).toHaveBeenCalledOnceWith('/currency/ip');
+        done();
+      });
+    });
+
+    it('should return `USD` if api does not return currency', (done) => {
+      apiService.get.and.returnValue(of(currencyIpData2));
+
+      orgService.suggestOrgCurrency().subscribe((res) => {
+        expect(res).toEqual('USD');
```
Comment (rvab) [inline src/app/core/services/org.service.spec.ts:121]: what happens on API failure?

**Review by rvab — APPROVED** [review]
minor comments/doubts

File: src/app/core/services/org.service.spec.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/core/services/org.service.spec.ts:95]: we are busting the cache before switching org

**Pull request merged**